src/arch/xtensa/configs/cannonlake_defconfig

```diff
@@ -5,3 +5,4 @@ CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=47
 CONFIG_PERFORMANCE_COUNTERS=y
+CONFIG_COMP_AEC=y
```

Comment:
do we want to change the defaults?

Reply:
I'll leave this out, this is now for my own convenience. I think AEC should be when merged default "n" until the algorithm is usable and mature. Even with with first adaptive filter and simple adaptation control in place I'd call it a mock-up. There will be need for VAD (voice activity detector) and residual echo conceal etc, speaker path AEC part for gain/stability control. Maybe multiple bands, maybe switch entirely to frequency domain with @keyonjie 's FFT library.